services/localega-tsd-proxy/src/main/resources/application.yaml

```diff
@@ -58,6 +58,12 @@ spring:
         username: ${TSD_MQ_USERNAME:admin}
         password: ${TSD_MQ_PASSWORD:admin}
 
+  security:
+    user:
+      name: ${ADMIN_USER:admin}  # Default user if not provided
```

Comment:
We should mot have preset defaults like this according to the Data Protection by Design and Defaults principles.

Desired effect: no access allowed at all, unless explicit userdefined username and password are provided at deployment time.

Reply:
Now the credentials are externalized and there's a check in place both in config level and application level.
By default spring will throw an error if the ADMIN_USER and ADMIN_PASSWORD is not set/present in the environment.

https://github.com/ELIXIR-NO/FEGA-Norway/pull/304/files#diff-251ee01c783145fba18f7c2462988d2150fdecee56319a9b5001c6b7f9f5348aR75-R84